package.json

```diff
@@ -2,7 +2,7 @@
     "name": "azure-pipelines",
     "displayName": "Azure Pipelines",
     "description": "Syntax highlighting, IntelliSense, and more for Azure Pipelines YAML",
-    "version": "1.157.1",
+    "version": "1.157.2",
```

Comment:
I think you need to `npm install` to get package-lock.json to take the new version number, too.

Reply:
package-lock.json has the updated version 1.157.2